src/XRP/xrp-client-decorator.ts

```diff
@@ -81,9 +81,21 @@ export interface XRPClientDecorator {
    * Note: This method only works for payment type transactions, see: https://xrpl.org/payment.html
    * Note: This method only returns the history that is contained on the remote node, which may not contain a full history of the network.
    *
-   * @param address: The address (account) for which to retrive payment history.
-   * @throws: An error if there was a problem communicating with the XRP Ledger.
-   * @return: An array of transactions associated with the account.
+   * @param address The address (account) for which to retrieve payment history.
+   * @throws An error if there was a problem communicating with the XRP Ledger.
+   * @return An array of transactions associated with the account.
    */
   paymentHistory(address: string): Promise<Array<XRPTransaction>>
+
+  /**
+   * Retrieve the payment transaction corresponding to the given transaction hash.
+   *
+   * Note: This method can return transactions that are not included in a fully validated ledger.
+   *       See the `validated` field to make this distinction.
+   *
+   * @param transactionHash The hash of the transaction to retrieve.
+   * @throws An error if the transaction hash was invalid.
+   * @return An XRPTransaction object representing an XRP Ledger transaction.
+   */
+  getPayment(transactionHash: string): Promise<XRPTransaction | undefined>
```

Comment:
Totally random question, why is this an interface as opposed to an `abstract class`?

(I'm not sure how this is actually used, so that may answer my question).

It seems weird that we have to JSDoc this same function in so many places. I wonder if there's a clean way to not do that. (Might be out of scope for this PR though).

Reply:
It hasn't been clear to me how to handle this problem either. I've been JSDoc'ing everything, since ultimately I'd like to generate documentation automatically from source. 

I'm curious what benefits you think an abstract class would have here?